src/dyad/core/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+set(DYAD_CORE_SRC ${CMAKE_CURRENT_SOURCE_DIR}/dyad_core.c)
+set(DYAD_CORE_PRIVATE_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/dyad_envs.h)
+set(DYAD_CORE_PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/dyad_core.h)
```

Comment:
If we don't want users directly calling DYAD Core, we should not have `dyad_core.h` in public headers. Alternatively, if we do want to install `dyad_core.h`, we need to add `dyad_dtl.h` to the public headers in the `dtl` directory.

Reply:
Fixed